### DISCUSSION TITLE
Property search button value displays all properties

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#452] [_Discarded pitches]
### Current State

The button value in the property search displays all properties (e.g. 2 of which one is a normal property and one is a reference property), although reference properties are marked as hidden in WP Backend.

E.g. https://testinstanz-fal.onofficeweb.com/immobilien/

### Desired State

The button value should show the correct amount of properties.

- If the references are marked as hidden in the backend, the button value should not display them. (See screenshots) Only the amount of normal properties should be displayed in the button value.
- If the references + Other properties are selected, they whole amount should be displayed in the button value.
- If only references are selected, only the amount of reference properties and no other properties should be displayed in the button value.

![references_disabled_backend](https://user-images.githubusercontent.com/88546396/219030072-bb10a4d4-89b0-44df-a1a9-6a7e3c7c2410.png)

![button_and_list_frontend](https://user-images.githubusercontent.com/88546396/219030067-949da132-0dd8-4e47-ba7d-bf1d79cf4fb3.png)

**Comment by fredericalpers** [comment]
@nglelinh @dai-eastgate Do you think this will be a fast and easy fix?

**Reply by dai-eastgate** [reply]
@fredericalpers After a quick investigation I thought it was easy to fix but right now I need to prioritize the issues in this cycle first.

**Reply by jmaas-onoffice** [reply]
@dai-eastgate Would you estimate that the fix takes more than three hours, including further investigation?

If it takes less, we would schedule it for ramp-up. If it takes more, we would schedule it for the next cycle. ;)

**Reply by dai-eastgate** [reply]
@jmaas-onoffice As far as I researched for 1 hour yesterday, I found fixing this issue can take 3 hours, so I think that we can do it during ramp up

**Comment by dai-eastgate** [comment]
I fixed this issue, you can watch the video and give me feedback.

https://user-images.githubusercontent.com/106214469/223637908-2669cefd-c56a-4c06-bcb9-354a54e5fb4b.mp4

**Reply by fredericalpers** [reply]
Unfortunately it seems like the issue raised is not fixed yet. In your video (00:03 - 00:04) you can see that when the references are hidden, the frontend still displays the number of estates wrong. Now they are displayed wrong in the search button as well as in the list itself.

**Reply by dai-eastgate** [reply]
@fredericalpers I checked again at (00:03-00:04). The estate list taken from my API has 11 properties (6 references + 4 normal + 1 top offer).
If I select the option "Hide reference estate" as the screen you see, only 5 normal properties are displayed in both the search button and the list.
So what does "displayed wrong in the search button as well as in the list itself" mean?

**Reply by fredericalpers** [reply]
Thank you for clarifying. The information on how many properites are from your API was helpful to understand. Seems fine to me then. Let me consult with our devs on how we will proceed from here.

**Reply by dai-eastgate** [reply]
fixed at https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/459 and code merged to master
